Sources/Testing/Running/Runner.Plan.swift

```diff
@@ -41,7 +41,7 @@ extension Runner {
       ///
       /// - Parameters:
       ///   - skipInfo: A ``SkipInfo`` representing the details of this skip.
-      indirect case skip(_ skipInfo: SkipInfo = .init())
+      indirect case skip(_ skipInfo: SkipInfo)
```

Comment:
This is a source-breaking tools SPI change. Can we keep the old signature but deprecate it? (We don't need ABI compatibility, just source, so a `public tatic func skip() -> Self` may be sufficient.)